README.md

```diff
@@ -129,6 +129,157 @@ allprojects{
 }
 ```
 
+## How to manually modify Metadata
+
+The Plugin makes it possible to manually add Manufacture-Data and Licenses-Data to the Metadata of the BOM. <br>
+The structure of the Metadata is shown on https://cyclonedx.org/docs/1.4/json/#metadata. <br>
+The editing of the Manufacture and Licenses-Data is optional. If the Manufacture/Licenses-Date isn't edited,
+then the respective structure won't appear in the BOM.
+
+To enable the modification of the metadata the cyclonedx-core-java plugin must be implemented in the build.gradle.
+
+---
+
+## Adding Manufacture-Data
+
+In order to be able to define the Manufacture-Data you must __import org.cyclonedx.model.*;__ into the build.gradle.
+<br>
+You can add the Manufacture-Data by passing an Object of the Type __OrganizationalEntity__ to the Plugin.
+
+__Example (groovy):__
+```groovy
+cyclonedxBom {
+    //declaration of the Object from OrganizationalEntity
+    OrganizationalEntity organizationalEntity_temp = new OrganizationalEntity();
+
+    //setting the Name[String] and Url[List] of the Object
+    organizationalEntity_temp.setName("Test");
+    organizationalEntity_temp.setUrls(["www.test1.com", "www.test2.com"]);
+    //declaration of the Object from OrganizationalContact
+    OrganizationalContact organizationalContact = new OrganizationalContact();
+    //setting the Name[String], Email[String] and Phone[String] of the Object
+    organizationalContact.setName("Max_Mustermann");
+    organizationalContact.setEmail("max.mustermann@test.org");
+    organizationalContact.setPhone("0000 99999999");
+    //adding the Object[OrganizationalContact] to the Object[OrganizationalEntity]
+    organizationalEntity_temp.addContact(organizationalContact);
+
+    //passing organizationalEntity_temp to the plugin
+    organizationalEntity = organizationalEntity_temp;
```

Comment:
We could create a custom DSL here to make it easier to configure the `organizationalEntity` and the `licenseChoice`. 

In the plugin, you can declare the `setOrganizationalEntity` as following 
````
public void setOrganizationalEntity(Consumer<OrganizationalEntity> customizer) {
     var origin = new OrganizationalEntity();
     customizer(origin);
     ...
}
````

Thus we could declare the configuration as 👍 

````
cyclonedxBom {
    organizationalEntity {
        name  'Max_Mustermann'
        email 'max.mustermann@test.org'
    }
}
````

and for Kotlin: 


````
tasks.cyclonedxBom {
    setOrganizationalEntity {
        name="Max_Mustermann"
        email="max.mustermann@test.org"
    }
}
````